backend/.env.example

```
@@ -46,6 +46,12 @@ CORS_ORIGIN="http://localhost:5174"
 # For Render: Use generateValue: true in render.yaml
 JWT_SECRET="your-secret-key-here"
 
+# Super-admin email — sole gatekeeper for user invite management
+SUPER_ADMIN_EMAIL="rlazarus@ssaandco.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)